packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -66,16 +66,16 @@ export class Breadcrumb extends BaseComponent<IBreadcrumbProps, IBreadcrumbState
   }
 
   public render() {
-    let { className } = this.props;
+    let { className, navigationLandmarkAriaLabel } = this.props;
```

Comment:
I would vote for `ariaLabel`.. and @petdun2519 does it have to be on the FocusZone or could it be on the surrounding `div`?